docs/products/opensearch/howto/upgrade-to-opensearch.rst

```diff
@@ -43,3 +43,58 @@ Extra information for Terraform users
 -------------------------------------
 
 If you manage your infrastructure with our :doc:`Terraform provider </docs/tools/terraform>` then please note that you will need to upgrade the provider to the latest release in order to have support for OpenSearch.
+
+To perform the upgrade in place using Terraform, you will need to take the following steps:
+
+1. Change the ``elasticsearch_version = 7`` to ``opensearch_version = 1``. This is the equivalent to clicking the migrate button in the console.
+
+.. code-block::
```

Comment:
Is it important to keep the block with `elasticsearch_version = 7`?  it is what user already have, right? Unless it gives information, I'd remove this block and only leave what user should have in the end.

Reply:
Or, do we want to clarify why there are two code snippets?

Reply:
Yes, the first block is what the user will already have if they manage their infrastructure with TF and I wanted to highlight the change that needs to be made to make the migration happen.